dev-libs/softhsm/softhsm-2.3.0.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="An implementation of a cryptographic store"
+DESCRIPTION="SoftHSM is an implementation of a cryptographic store \
+accessible through a PKCS #11 interface. You can use it to explore PKCS #11 \
+without having a Hardware Security Module. It is being developed as a part \
+of the OpenDNSSEC project. SoftHSM uses Botan for its cryptographic \
+operations."
+HOMEPAGE="https://www.opendnssec.org/softhsm/"
+COPYRIGHT="2010 .SE, The Internet Infrastructure Foundation and SURFnet bv"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://dist.opendnssec.org/source/softhsm-$portVersion.tar.gz"
+CHECKSUM_SHA256="5ed604c89a3a6ef9d7d1ee92c28a2c4b3cd1f86f302c808e2d12c8f39aa2c127"
+SOURCE_DIR="softhsm-$portVersion"
```

Comment:
No need as `SOURCE_DIR` defaults to `softhsm-$portVersion` anyway

```diff
-SOURCE_DIR="softhsm-$portVersion"

Reply:
Good catch! Thanks @alaviss!

Reply:
The messages above are outdated.
(We do need the `SOURCE_DIR` because we have chosen to rename the recipe.)